metadata-io/src/main/java/com/linkedin/metadata/graph/Neo4jGraphService.java

```diff
@@ -99,7 +99,7 @@ public void addEdge(@Nonnull final Edge edge) {
 
     String statementString =
         String.format(matchTemplate, sourceType, srcCriteria, relationshipTypeFilter, edgeCriteria,
-            destinationType, destCriteria);
+            destinationType, destCriteria, sourceType);
```

Comment:
Should we be returning destinationType?

Reply:
@jjoyce0510 I believe you are right. At this point I am unable to recall, what was the reason for `sourceType` here versus `destinationType`. The `destinationType` looks expected. Let me change it

Reply:
Done